koku/masu/util/ocp/ocp_post_processor.py

```diff
@@ -163,7 +164,41 @@ def _generate_daily_data(self, data_frame):
 
         return daily_data_frame
 
-    def process_dataframe(self, data_frame):
+    def _remove_anomalies(self, data_frame: pd.DataFrame, filename: str) -> pd.DataFrame:
+        """Removes rows with anomalous values from the DataFrame."""
+
+        threshold_map = {
+            # 1,000,000,000,000,000,000 is a resonable value that should remove most anomlies from bad data
+            1e18: [
+                "pod_usage_cpu_core_seconds",
+                "pod_request_cpu_core_seconds",
+                "pod_limit_cpu_core_seconds",
+                "node_capacity_cpu_core_seconds",
+                "pod_usage_memory_byte_seconds",
+                "pod_request_memory_byte_seconds",
+                "pod_limit_memory_byte_seconds",
+                "node_capacity_memory_byte_seconds",
+                "node_capacity_memory_bytes",
+                "persistentvolumeclaim_capacity_bytes",
+                "persistentvolumeclaim_capacity_byte_seconds",
+                "volume_request_storage_byte_seconds",
+                "persistentvolumeclaim_usage_byte_seconds",
+            ],
+        }
+        thresholds = {col: thresh for thresh, cols in threshold_map.items() for col in cols}
```

Comment:
Can we make this a constant somewhere outside of this func? Otherwise we re-create this map everytime this function is called, which is quite frequently throughout the day.